sklearn/preprocessing/data.py

```diff
@@ -503,11 +511,10 @@ class StandardScaler(BaseEstimator, TransformerMixin):
 
     See also
     --------
-    :func:`sklearn.preprocessing.scale` to perform centering and
-    scaling without using the ``Transformer`` object oriented API
+    scale: Equivalent function without the object oriented API
 
-    :class:`sklearn.decomposition.RandomizedPCA` with `whiten=True`
-    to further remove the linear correlation across features.
+    :class:`sklearn.decomposition.PCA`
+        Further removes the linear correlation across features with 'whiten=True'.
```

Comment:
Similar to above comment [here](https://github.com/scikit-learn/scikit-learn/pull/6748/files#r61790813). These are from the current docs of `StandardScaler`.
![screenshot 2016-05-02 14 38 11](https://cloud.githubusercontent.com/assets/1865885/14965543/9b970e94-1073-11e6-8317-cb6f13b7f7ca.png)